discord_bot/bot.py

```diff
@@ -0,0 +1,88 @@
+import os
+from dotenv import load_dotenv
+import asyncio
+from interactions import (
+    AutocompleteContext,
+    Client,
+    Intents,
+    listen,
+    slash_command,
+    SlashContext,
+    SlashCommand,
+    slash_option,
+    SlashCommandChoice,
+    OptionType,
+)
```

Comment:
In shared project, it's better to use module namespace. So preferably avoid importing the function in. What happen if you have OpenAI.Client for example. It's better to be explicit to avoid any confusion.

Use `@interactions.listen()` instead of plain `listen()` is best practice.